src/stream/src/executor/hash_agg.rs

```diff
@@ -326,11 +327,12 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
     ) {
         // The state store of each keyspace is the same so just need the first.
         let store = keyspace[0].state_store();
+        let table_id = keyspace[0].table_id();
         // --- Flush states to the state store ---
         // Some state will have the correct output only after their internal states have been
         // fully flushed.
         let (write_batch, dirty_cnt) = {
-            let mut write_batch = store.start_write_batch();
+            let mut write_batch = store.start_write_batch(WriteOptions { epoch, table_id });
```

Comment:
I'm afraid this is not correct. HashAgg uses a lot of tables, we should produce multiple write batches in hash agg.

Reply:
Right both hash_agg.rs and global_simple_agg.rs have been refactored after this PR. Will fix.

Reply:
fixed